modeling/meters.md

```diff
@@ -0,0 +1,132 @@
+In fact, meters can *only* be related to each other via these relationships.
+
+```ttl
+:building_meter a brick:Building_Meter ;
+    brick:hasSubMeter :floor-meter-1, :floor-meter-2 .
+:floor-meter-1 a brick:Meter .
+:floor-meter-2 a brick:Meter .
+```
+
+```{warning}
+Brick does not add any properties to the submeter relationship --- it only indicates that there is one. If this is a problem, [let us know!](https://github.com/BrickSchema/Brick/issues)
+```
+
+To ask for the immediate submeters of a particular meter, one can query the model as follows:
+
+```sparql
+SELECT * WHERE {
+    :building_meter brick:hasSubMeter ?submeter .
+}
+```
+
+or more generally, for all immediate parent/child submeter relationships:
+
+```sparql
+SELECT ?meter ?submeter WHERE {
+    ?meter rdf:type/rdfs:subClassOf* brick:Meter .
+    ?meter brick:hasSubmeter ?submeter .
+}
+```
+
+One can also ask for all meters that *aren't* submeters of any other meter. Assuming the Brick model is correct, this will avoid double counting:
+
+```sparql
+SELECT DISTINCT * WHERE {
+    ?meter rdf:type/rdfs:subClassOf* brick:Meter .
+    FILTER NOT EXISTS { ?meter brick:isSubMeterOf ?parent }
+}
+```
+
+## Associating Meters
+
+Meters can be associated with the entities that they are metering through the `brick:meters` / `brick:isMeteredBy` relationships. These entities can be instances of `brick:Equipment`, `brick:Location` *or* `brick:Collection`.
+
+Metering `brick:Equipment`: useful when the Meter is measuring the consumption/production of a single Equipment instance.
+
+```ttl
+:chiller-1 a brick:Chiller ;
+    brick:isMeteredBy :chiller-meter .
+:chiller-meter a brick:Electric_Meter .
+```
+
+Metering `brick:Location`s: useful when the Meter is measuring the consumption/production of many entities within a single location, e.g. a building or floor or room.
+
+```ttl
+:floor-1 a brick:Floor ;
+    brick:isLocationOf :camera-1, :camera-2 ;
+    brick:isMeteredBy :floor-meter .
+:floor-meter a brick:Electric_Meter .
+:camera-1 a brick:Camera .
+:camera-2 a brick:Camera .
+```
+
+Metering `brick:Collection`s: useful when the Meter is measuring the consumption/production of many entities that are not necessarily grouped by location. Recall that a `brick:Collection` is a named group of entities. This could represent the equipment in a particular air loop, in a particular chilled water system, all of the devices owned by a particular individual, or any other organization.
+
+```ttl
+# pv generation systems and pv_arrays are both collections
+:pv_generation_system a brick:PV_Generation_System ;
+    brick:hasPart :array1 ;
+    brick:isMeteredBy :pv_meter .
+site:array1 a brick:PV_Array .
+site:panel11 a brick:PV_Panel ;
+    brick:isPartOf site:array1 ;
+    brick:panelArea [ brick:hasUnit unit:M2 ;
+            brick:value "5"^^xsd:double ] .
+site:panel12 a brick:PV_Panel ;
+    brick:isPartOf site:array1 .
+:pv_meter   a   brick:Electrical_Meter .
+```
+
```

Comment:
An example query that works for all of those patterns would be helpful.

Reply:
Good idea! Added one